apps/dashboard/config/application.rb

```diff
@@ -1,13 +1,19 @@
-require File.expand_path('../boot', __FILE__)
+require_relative 'boot'
 
-require "rails"
-# Pick the frameworks you want:
-require "active_model/railtie"
-require "active_job/railtie"
+# require "rails"
+
+# This is not loaded in rails/all but inside active_record so add it if
+# you want your models work as expected
+require "active_model/railtie" 
+require "active_job/railtie"  # New in 5.2.x
 # require "active_record/railtie"
+
+# And now the rest
 require "action_controller/railtie"
 require "action_mailer/railtie"
 require "action_view/railtie"
+require "action_cable/engine"
```

Comment:
Should we omit `action_cable/engine` till we actually use it?

Reply:
Done.